test/functional/ws/transport/transactions.js

```diff
@@ -30,63 +30,41 @@ function postTransaction(transaction, cb) {
 		{
 			transactions: [transaction],
 		},
-		cb,
+		() => {},
 		true
 	);
+	cb();
 }
 
 describe('Posting transaction (type 0)', () => {
-	var account;
-	var transaction;
-	var error;
-	var response;
-	var goodTransactions = [];
-	var badTransactions = [];
+	let transaction;
+	const goodTransactions = [];
+	const badTransactions = [];
+	const account = randomUtil.account();
 
 	beforeEach(done => {
-		account = randomUtil.account();
 		transaction = randomUtil.transaction();
 		done();
 	});
 
-	describe('when sender has no funds for a transaction in batch', () => {
-		beforeEach(done => {
+	describe('transaction processing', () => {
+		it('when sender has no funds should fail', done => {
 			transaction = lisk.transaction.createTransaction(
 				'1L',
 				1,
 				account.password
 			);
-			postTransaction(transaction, (err, res) => {
-				error = err;
-				response = res;
+
+			postTransaction(transaction, () => {
```

Comment:
Transaction is no longer pushed to `badTransactions`, so what are we testing exactly?

Reply:
Thanks, I've added both transactions to good and bad lists.